docs/openapi/schemas/VerifiablePresentation.yml

```diff
@@ -10,9 +10,23 @@ allOf:
         $ref: "./PresentationLinkedDataProof.yml"
 example:
   {
-    "@context": ["https://www.w3.org/2018/credentials/v1"],
+    '@context': [
+      'https://www.w3.org/2018/credentials/v1',
+      'https://w3id.org/traceability/v1'
+    ],
     "id": "urn:uuid:07aa969e-b40d-4c1b-ab46-ded252003ded",
-    "type": ["VerifiablePresentation"],
+    'type': [
+      'VerifiablePresentation',
+      'TraceablePresentation'
```

Comment:
This schema should not be a "Traceable Presentation" it should match normative requirements of the W3C TR.